### ISSUE TITLE
Document the research coordination processes per interview

### ISSUE THREAD

**Issue by Baleja** [codeforamerica/brigade-playbook#122]
### Overview
We need a record of the process once the interview is completed to handing off to the UXR.

### Action Items
Each person to define the process. 
- [ ] @MarianneAnthonette
- Start the process once a Calendly new event e-mail has been received in the BOP Gmail.

1. Update tracking sheets:
   - In the BOP Interview Responses and Tracking sheet, navigate to the Interview Response List + Tracking tab and add the new interview information (e.g. brigade name, brigade region, brigade lead name(s), interview status -> 'In Progress' into the next available row.
   - In the UXR Schedule sheet, add the interview info (e.g. brigade name, brigade lead's name, brigade lead timezone) to the appropriate Date of Interview row.
      - Change column A to 'Available for interviewee'
   - In the Brigade Contact Directory sheet, go to the Contact Status Brigades tab and update the status to 'Scheduled'. (This step only applies to Brigades who haven't already participated.)
2. Find an interviewer for the interview:
   - Post the available interview on Slack with the following message format:
      - 📢   There is an interview up for grabs <day_of_week>, <month/day> @<interview_time> PT. If you are interested, please comment!
   - Before sending the message, tag interviewer volunteers onto the message.
3. Once the interview has an interviewer:
   - Update the interview post on Slack by applying a strikethrough to the message and adding a comment saying that it's taken.
   - Add the interviewer onto the Google Calendar invite for the interview.
   - Update the interview info with the interviewer's name (column F) in the UXR schedule.
      - Check if the prefilled Google form link works. If not, make sure that column L contains the interviewer's name.
4. On the day of the interview:
   - If the interviewer doesn’t post a check-in message in the #organizers-playbook Slack channel at least 10 minutes prior to the interview, send a direct message to them and be on stand-by for the interview.
   - Once the interview is complete, go to the Interview Responses and Tracking sheet and change the interview status to 'Complete'.
   - If the interview has multiple interviewees, send an interview thank-you email with a pdf copy of the form responses to the second interviewee (the interviewee whose name isn't included in the Calendly event name).

- @thadk instruction 1

- Share OtterAI contents with the team: Changing the title of the otter.ai, make a sharable link and share to BOP group.
       - Whoever has the main paid OtterAI credentials needs to add any new interview to the OtterAI BOP sharing group, so other team members can access it with their unpaid otterAI accounts.
       - That paid OtterAI user should set the OtterAI interview title to match the audio format exactly. That format is "Rename the recording with the YourUXR#, the name of the brigade you have interviewed, the date of the interview (year-month-day) and the name of the person you interviewed." `Example 1: 007_hackforla_2020-09-28_Bonnie-Wolfe`
& `Example 2: 003_codeforcharlottesville_2020-10-22_Jonathan-Kropko`.
      - That paid OtterAI user should enable link sharing for that interview (**only if** everyone involved has agreed to be careful with the links!).
 
- Match Speakers: Anyone on the OtterAI group
      - Mark speakers in OtterAI: We used "First Name, last initial, (Brigade Name)" for interviewees. We used "Firstname Lastname" for interviewers.
      - Find and open the OtterAI sheet of the [interview responses and tracking sheet](https://docs.google.com/spreadsheets/d/10uZjjrXpZH-y6wZBHZvhKgWNUBj7Xvyh9yoiqL4F_9I/edit#gid=1930219941&fvid=48587054).
      - Add new rows for each new speaker and write whether they're an interviewer or interviewee.

- [ ] @raquelnish - instruction 1
- Start the process when Otter.ai sends an email informing a new interview is done. 
- Create transcription spreadsheet following instructions of  item #62 
- At the BOP Interview Responses and Tracking sheet> Interview Response list+ tracking tab:
1. Add information (formula) on every column: 
     - Target Row
     - Transcription Status
      - Shared effective processes/practices during the interview 
      - Practices added to assets spreadsheet (internal)
      - Interest in self-service version
      - People to follow-up with (cross-off when contacted)
      - How public?
      - Brigade Size
- At the BOP Interview Responses and Tracking sheet> Otterai tab: 
      - Add interviewee name to column A as it is formatted on Otter.ai / on column B write interviewee
      - Check if the Interviewer´s name is on the list, if it is not, add it to column A as it is formatted on Otter.ai/ on column B write interviewer.
      
- @thadk instruction 2

2.  Integrate first-pass OtterAI contents to transcript sheet:
      - Copy the OtterAI link to the interview into the interview's Transcription Sheet (from earlier step, around #62).
      - Export out a txt file from OtterAI to the `Transcription Sheets/txtfiles`
      - Run the [interviewclean.ipynb script in the Research Analysis](https://drive.google.com/file/d/11zPCJjkXuI5GwiegRU8qXK8VMqdC0oI1/view?usp=sharing) folder. This script drops dialog from the interviewer and find the dividing points for roughly 13 sections of the interviewee. For the divisions it find, it formats these sections into a "Transcript Dump" google sheet in the `Transcription Sheets/txtfiles` folder. It **should** also get pulled into the "Auto Parsed OtterAI text data" referenced in the next step.
      - Check to make sure the [interview responses and tracking sheet](https://docs.google.com/spreadsheets/d/10uZjjrXpZH-y6wZBHZvhKgWNUBj7Xvyh9yoiqL4F_9I/edit#gid=1930219941&fvid=48587054) on sheet for "Interview Response List + Tracking" – column J "Interview ID" and column K "Transcription spreadsheet link" must be populated for the script to work.
      - If you have trouble with the script, check that "Auto Parsed OtterAI text data" sheet has updated with your last interview.
      - Run the ["Transcription Merging tasks in Google Sheets" script in the "Research Analysis" folder](https://colab.research.google.com/drive/1NxAf6W1N1JFC0w1ZdekCHjWKlfQo6Rif?usp=sharing) to load the otter.ai into the transcription sheet (was #49)
      - The Transcript Sheet should now have prefilled transcripts from OtterAI in the second column.

- @raquelnish - instruction 2

2. Create a GitHub issue
    - At the UXR Management Team Board  create a new issue using the ‘Interview Priority Analysis’ issue template and edit it:
    - Add the interviews Target Row Number (find it at BOP Interview Responses and Tracking sheet> Interview Response list+ tracking tab column H)  at "Number of Interview" both on the issue title and at the first line of "Overview".
    - Add the transcription spreadsheet link to [Interview Data Sheet][insert url] at the bottom of this issue in the Resources/Instructions area.
    - Label it as ‘Transcription Verification’.
    - Add it to the ‘UXR Team Management’ project.
    - As Milestone, choose number ‘7 - Analysis, Synthesis & Report Creation.
    - Put it into the ‘In progress’ box.
    - Add this issue number as a link to the BOP Interview Responses and Tracking sheet> Interview Response list+ tracking tab, issue number column I.

3. At the At the BOP Interview Responses and Tracking sheet> Interview Response list+ tracking tab: If  the answer to "Shared effective processes/practices during the interview" is Yes:
      - Fill up the BOP Processes and Practices Data sheet >DATA ENTRY tab, with information from BOP Interview Responses and Tracking sheet>Form Response 1 tab  starting at column Z
      - If a Brigade has multiple assets, put one in each row
      - When adding a link to google doc/spreadsheet/slide create a preview and copy links by  opening the url and:
      Ex. ://docs.google.com/document/d/1-kVIfpPg4EW0GzG2cQp8hunNOaCMejy2ZAKjnac7AF8/ ~~edit~~
          - Preview mode: → Replace /edit with /preview 
          - Force a copy: → Replace /edit with /copy
      - At column "Make resource public": Only mark as 5 if the resource really offers clear instructions ready to use. If it does not, even if it is marked as 5 at usability, you can grade it 4 or below accordingly.
      -  If the "Asset type" column has assets in it but no assets links, mark 1 at the column"Follow up to get links"
      - Go back to At the BOP Interview Responses and Tracking sheet> Interview Response list+ tracking tab: Mark as "Yes" the column "Practices added to assets spreadsheet (internal)"

### Resources/Instructions
[Research coordination instructions folder](https://drive.google.com/drive/u/1/folders/1fjav4S1_YO8s6-4-K6JEHzdGQJt8eQmT)
[Interview Scheduling](https://docs.google.com/document/d/1cMJIZpYGqWACtk7zEoS5Pj5RKwiUioVj_agzZKKICyc/edit#)
[OtterAI Set-up](https://docs.google.com/document/d/1OfbbleSugZegpgblxL6xC8v_sUGIS58T8X13RRC4VQ0/edit#)
[Create and Update 1: Transcription Sheet](https://docs.google.com/document/d/1ZSo1-XfC6gpXT6snyw7UcEororeaZN-lj4UfptcsJ8k/edit?usp=drive_web&ouid=116699334996809624012)
[Pre-fill Transcription Sheets](https://docs.google.com/document/d/1hdOMIFVPTwg_1PmSeUhg-BTpw4HXO2aVDflLXNY754M/edit#)
[Create and Update 2: GitHub Issue and P&P](https://docs.google.com/document/d/1peMLcR1PvPstgctPQgVemS8bD10olch_7-Y7fxY_P58/edit)

**Comment by Baleja** [comment]
@Thad to document steps on issue 122 - Document the research coordination processes per interview

**Comment by Baleja** [comment]
@raquelnish & team will make a new template for the scheduling, transcribing, documentation & other processes : 
 "Procedure for UXR scheduling and interviewing prior to analysis"

**Comment by Baleja** [comment]
Progress: Thad updated his steps above, the sections need to be cleaned up and this issue made into a template. Headers need to be made to break up the sections.
Blockers: none
ETA: Tuesday 12th Jan

**Comment by Baleja** [comment]
@Baleja Make a google document for each persons section with thier instructions under thier name. (some might need two docs or two sections in the doc depending on handoff.

**Comment by Baleja** [comment]
Progress: We are migrating instructions to the Google drive, we will create a separate issue with clear action items. @thadk & @MarianneAnthonette adding to thier google docs this week.
Blockers: No blocks 
Availability:
ETA:

**Comment by raquelnish** [comment]
BOP Documentation 1:
- [ ] Start the process with OtterAI 
- [ ] Use template to create new transcription spreadsheet
- [ ] Update Tracking sheet

BOP Documentation 2:
- [ ] Create a GitHub issue
- [ ] Update Tracking sheet
- [ ] Add Process & Practices
- [ ] Update Tracking sheet





## Resources ##

[BOP Documentation1 [INTERNAL]](https://docs.google.com/document/d/1ZSo1-XfC6gpXT6snyw7UcEororeaZN-lj4UfptcsJ8k/edit#heading=h.8dt2qeurfmti)
[BOP Documentation 2 [INTERNAL]](https://docs.google.com/document/d/1peMLcR1PvPstgctPQgVemS8bD10olch_7-Y7fxY_P58/edit#heading=h.h0fnvfy3o5vh)

**Comment by MarianneAnthonette** [comment]
My very rough proposition for the Action Items for the issue template we will make. The names of the steps and labels are just examples but I think it is a good start.

### Action Items

- [ ] This brigade interview needs the following steps done:
     - [ ] 1. Interview Scheduling
          - [ ] Add OtterAI Set-Up label & remove Interview Scheduling label.
     - [ ] 2. OtterAI Set-Up
          - [ ] Add Create and Update 1 label & remove OtterAI Set-Up label.
     - [ ] 3. Create and Update 1: Transcription Sheet 
          - [ ] Add Pre-fill Transcription Sheets label & remove Create and Update 1 label.
     - [ ] 4. Pre-fill Transcription Sheets
          - [ ] Add Create and Update 2 label & remove Pre-fill Transcription Sheets label.
     - [ ] 5. Create and Update 2: GitHub Issue and P&P

**Comment by Baleja** [comment]
Progress: Raquel & Mari have completed thier instructions
Blockers: None
Availability:
ETA: next week 19th/22nd

**Comment by MarianneAnthonette** [comment]
All instruction documents have been created. (Mostly) ready to create the template issue. Might need some last-second renaming/formatting of instructions once we create the template issue.

**Comment by Baleja** [comment]
Drafted this as the Template Issue, can we review this together @thadk @MarianneAnthonette @raquelnish

### Overview
Complete processes for conducting User interviews & processes for analyzing. 

### Instructions for working this issue
Do the action items defined by the checkboxes. Each numbered step is a discrete task, which is outlined more in the correlated instructions. 

### Action Items
##[Setting up the interview](https://docs.google.com/document/d/1OfbbleSugZegpgblxL6xC8v_sUGIS58T8X13RRC4VQ0/edit?usp=sharing)
- [ ] This brigade interview needs the following steps done:
     - [ ] 1. Interview Scheduling
          - [ ] Add OtterAI Set-Up label & remove Interview Scheduling label.
     - [ ] 2. OtterAI Set-Up
          - [ ] Add Create and Update 1 label & remove OtterAI Set-Up label.
     - [ ] 3. Create and Update 1: Transcription Sheet 
          - [ ] Add Pre-fill Transcription Sheets label & remove Create and Update 1 label.
     - [ ] 4. Pre-fill Transcription Sheets
          - [ ] Add Create and Update 2 label & remove Pre-fill Transcription Sheets label.
     - [ ] 5. Create and Update 2: GitHub Issue and P&P
     - [ ] Share OtterAI contents with the team: Changing the title of the otter.ai, make a sharable link and share to BOP group.
- [ ] Whoever has the main paid OtterAI credentials needs to add any new interview to the OtterAI BOP sharing group, so other team members can access it with their unpaid otterAI accounts.
- [ ] That paid OtterAI user should set the OtterAI interview title to match the audio format exactly. That format is "Rename the recording with the YourUXR#, the name of the brigade you have interviewed, the date of the interview (year-month-day) and the name of the person you interviewed." Example 1: 007_hackforla_2020-09-28_Bonnie-Wolfe
& Example 2: 003_codeforcharlottesville_2020-10-22_Jonathan-Kropko.
-  [ ] That paid OtterAI user should enable link sharing for that interview (only if everyone involved has agreed to be careful with the links!).
- [ ]Mark speakers in OtterAI: We used "First Name, last initial, (Brigade Name)" for interviewees. We used "Firstname Lastname" for interviewers.
-  [ ]Find and open the OtterAI sheet of the interview responses and tracking sheet.
-  [ ]Add new rows for each new speaker and write whether they're an interviewer or interviewee.
 
##[Transcription sheet set up](https://docs.google.com/document/d/1ZSo1-XfC6gpXT6snyw7UcEororeaZN-lj4UfptcsJ8k/edit#heading=h.umhaeuaepkun)
- [ ] Start the process with OtterAI 
- [ ] Use template to create new transcription spreadsheet
- [ ] Update Tracking sheet

##[Interview progress tracking](https://docs.google.com/document/d/1peMLcR1PvPstgctPQgVemS8bD10olch_7-Y7fxY_P58/edit)
- [ ] Create a GitHub issue
- [ ] Update Tracking sheet
- [ ] Add Process & Practices
- [ ] Update Tracking sheet

     
### Reporting progress
- Be sure to check the box when you finish an action item.
- Report your progress weekly in a comment below using the following format:
     - Progress: 
     - Blockers: 
     - Availability :
     - Estimated date of completion:

### Stopping mid issue 
#### Reasons: 
- Got personally busy, can’t finish
-  Only want to do a specific type of task per interview
- There is a blocker (e.g. instructions not ready yet) in the way of finishing and I still have time to work on other issues 
#### What to do if you have to stop working mid issue:
- Note your progress in the checkboxes 
- Add a note in the comments with details
- Move the issue to the questions/review column 

### Resources/Instructions
[Interview Data Sheet](https://docs.google.com/spreadsheets/d/1p1Arx4ZFB9ixCP81FmkNRAK8iiiEtsgab67gjiVbp6o/edit#gid=1783476144)
[Transcription Verification Instructions](https://docs.google.com/document/d/1kTrslqYY0whUjCMxbcV7Ft1nzCr_UTPWXfhomNgOvEw/edit)
[Data Organization Instructions](https://docs.google.com/document/d/1vJqDQ8Th8kSgisSSiMzDpRHTIWI3ILCX4eOTgMwRdoY/edit)
[Coding Instructions](https://docs.google.com/document/u/0/d/10am6Ki0gJKzITt2kcjMsR233oC_9WJneKnFimFc8Mao/edit)
[Tagging Instructions](https://docs.google.com/document/d/1soNmLMyVspkgz2MMD5DuF_WVBBmnFLFgCPNZDjxjDWI/edit#)

**Comment by Baleja** [comment]
Progress: We have everything to make the template issue
Blockers: none

- [ ] @Baleja will make the template issue from Maris action item list. - Make it as simple as the previous
- [ ] Make labels for the issue

**Comment by Baleja** [comment]
Label the new labels as "RC" research co-ordination.

**Comment by Baleja** [comment]
https://www.awesomescreenshot.com/video/2489719

**Comment by Baleja** [comment]
Change the instructions for the action items with @maris action items and add the appropriate documentation. LABELS

**Comment by Baleja** [comment]
Labels IP for interview prep

**Comment by ExperimentsInHonesty** [comment]
@thadk The team wants to discuss with you at the next meeting 

To be added to instructions - add quantifiable size, region, length of brigade leadership.
1 year = 1 
6 months = 0.5
4 months = 0.5
